packages/IMAPClient-Core.package/ICEndPoint.class/instance/login.st

```diff
@@ -18,6 +18,6 @@ login
 		self fatalErrorOccured: true.
 		^ false].
  
-	self loadEmailsFromMaildir.
+	(ICFileReadWriter newWith: self) readEmailsFromDisk.
```

Comment:
Is there anything similar for the closing of the IMAP client to write them down?
Or how do they become persistent?

Reply:
They get written down when they get fetched.